README.md

```diff
@@ -1 +1,436 @@
-# pytorch-lightning-feedstock
\ No newline at end of file
+<div align="center">
```

Comment:
Don't add this README from pytorch lightning. Eventually we will have OpenCE README files.

Reply:
yeah it is copy of upstream - https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/0.9.0/README.md. 
Shall I remove it?

Reply:
re-read, I will remove it.